lib/OriginalSource.js

```diff
@@ -9,10 +9,10 @@ var SourceMapConsumer = require("source-map").SourceMapConsumer;
 var SourceListMap = require("source-list-map").SourceListMap;
 var Source = require("./Source");
 
-var SPLIT_REGEX = /(?!$)[^\n\r;{}]*[\n\r;{}]*/g;
+var SPLIT_REGEX = /([^\d\w]+)/g;
```

Comment:
Could you measure how much this affects performance? As far as I remember this could have negative performance impact on larger files.